tools/android/packaging/xbmc/src/Splash.java.in

```diff
@@ -286,8 +285,8 @@ public class Splash extends Activity
       }
       fXbmcHome.mkdirs();
 
-      // Log.d(TAG, "apk: " + sPackagePath);
-      // Log.d(TAG, "output: " + sXbmcHome);
+      Log.d(TAG, "apk: " + sPackagePath);
+      Log.d(TAG, "output: " + sXbmcHome);
```

Comment:
is there actually value in this being logged?

Reply:
Only logs them when it's installed for the first time, not on next boots. Activated them in case there is a problem with another device so that we can see the initial paths.